lib/mayaUsd/ufe/UsdShaderAttributeHolder.cpp

```diff
@@ -89,13 +91,34 @@ std::string UsdShaderAttributeHolder::defaultValue() const
 
 std::string UsdShaderAttributeHolder::nativeType() const { return _sdrProp->GetType(); }
 
+//! A List of the properties that are known to lack the default value.
+//! Insert the name of the property to avoid logging the warning.
+static const std::unordered_set<std::string> s_ListPropertiesWithoutDefaultValue {
+    "attenuation_distance", // shader: gltf_pbr
+    "intensity",            // shader: directional_light, point_light
+    "inner_angle",          // shader: spot_light
+    "outer_angle"           // shader: spot_light
+};
+
 bool UsdShaderAttributeHolder::get(PXR_NS::VtValue& value, PXR_NS::UsdTimeCode time) const
 {
     if (isAuthored()) {
         return _Base::get(value, time);
     }
     // No prim check is required as we can get the value from the attribute definition
     value = vtValueFromString(usdAttributeType(), defaultValue());
+
+    if (defaultValue().empty()) {
+        const auto propertyName = _sdrProp->GetName().GetString();
+        bool       isKnownProperty = s_ListPropertiesWithoutDefaultValue.find(propertyName)
+            != s_ListPropertiesWithoutDefaultValue.end();
+        if (!isKnownProperty) {
+            const std::string msg
+                = "Warning: Shader property '" + propertyName + "' does not have a default value.";
+            UFE_LOG(msg.c_str());
```

Comment:
This is where you add a new entry to s_ListPropertiesWithoutDefaultValue. Please include the shader info:id in both the message and in the entry because we would like to know the exact identity of the problematic shader.